content/en/docs/tasks/administer-cluster/upgrade-downgrade/kubeadm-upgrade-1-11.md

```diff
@@ -0,0 +1,303 @@
+---
+reviewers:
+- pipejakob
```

Comment:
nix all these and use the alias 
`- sig-cluster-lifecycle`

Reply:
I really should get https://github.com/kubernetes/website/pull/8506 in after all the docs flux of the release